server.js

```diff
@@ -1,5 +1,5 @@
 const app = require('./api/app.js');
 
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 8000;
```

Comment:
This is okay, but if you want to change the port in the future, you can do so in the .env file.

Reply:
I didn't do any of these changes

Reply:
Oh, I see this does match the main version. My bad

Reply:
Gotcha. Well, since this doesn't match what's in main right now, I'd like you to run `git checkout origin/main README.md` to restore this file to its main version.